try/try_stack_test.go

```diff
@@ -0,0 +1,54 @@
+package try_test
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/spacemonkeygo/errors"
+	"github.com/spacemonkeygo/errors/try"
+)
+
+func TestStackHandling(t *testing.T) {
+	// Crappy test.  Try harder later.
```

Comment:
This is a super terrible test.  I could say I left it in so it could spark some discussion, but really, I just forgot it was here.

I'd love suggestions for how to get coverage or describe this behavior correctly... otherwise it should probably thrown away, because all it does is print stuff that has to be eyeballed, and that's just not a test.